beacon/goclient/voluntary_exit.go

```diff
@@ -2,9 +2,8 @@ package goclient
 
 import (
 	"github.com/attestantio/go-eth2-client/spec/phase0"
-	"github.com/pkg/errors"
 )
 
-func (gc *goClient) SubmitVoluntaryExit(voluntaryExit *phase0.SignedVoluntaryExit, sig phase0.BLSSignature) error {
-	return errors.New("not implemented")
+func (gc *goClient) SubmitVoluntaryExit(voluntaryExit *phase0.SignedVoluntaryExit, _ phase0.BLSSignature) error {
```

Comment:
so maybe this can be `SubmitVoluntaryExit(voluntaryExit *phase0.SignedVoluntaryExit) error` ? of course might require spec change so only do when and if @GalRogozinski approves.

Reply:
@y0sher I already requested to change this. @MatheusFranco99 has a PR for this: https://github.com/bloxapp/ssv-spec/pull/319

Reply:
https://github.com/bloxapp/ssv-spec/pull/319 was merged, but I think we need to do this change on the next alignment with spec to avoid delaying this PR

Reply:
@Tomblox1 we might need a task here to not forget to align this once we align the spec.

Reply:
@y0sher I think we won't be able to forget to align this because the code will not compile after updating spec version